packages/app/src/composables/useTransaction.ts

```diff
@@ -130,9 +129,9 @@ export default (context = useContext()) => {
           amountPaid: transactionDetails.fee.toString(),
           isPaidByPaymaster: false,
           refunds: [],
-          amountRefunded: BigNumber.from(0).toHexString(),
+          amountRefunded: `0x${BigInt(0).toString(16)}`,
```

Comment:
Same

Reply:
Added an util function and used it in places like this.